compiler_gym/envs/cgra/Operations.py

```diff
@@ -0,0 +1,54 @@
+
+class Operation(object):
+    def __init__(self, name, inputs, outputs, latency):
+        self.name = name
+        self.inputs = inputs
+        self.outputs = outputs
+        self.latency = latency
+
+    def __str__(self):
+        return self.name
+
+Operations = [
+    # TODO --- should we support more operations as heterogeneous?
```

Comment:
Nitpick: attribute your TODO's so that we can keep tabs of who needs to do what. E.g.:

```suggestion
    # TODO(j-c-w) --- should we support more operations as heterogeneous?
```

(this probably applies elsewhere in the PR)